test/preprocessor/dm_option.f90

```diff
@@ -0,0 +1,90 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+! Just check some of the macros - if they are printed
```

Comment:
I suggest moving the blank line from line 1 to just before this line, to separate the copyright header from the comment visually.